test-libbz2-rs-sys/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "test-libbz2-rs-sys"
 readme = "README.md"
 # description.workspace = true
-# version.workspace = true
+version.workspace = true
 edition.workspace = true
 # license.workspace = true
 # repository.workspace = true
```

Comment:
Pre-existing, but this crate should probably be `publish = false`.

Reply:
fixed that up, but yes this crate is never supposed to be published, or used by anyone but us